pkg/chunk/grpc/gRPC.proto

```diff
@@ -0,0 +1,121 @@
+syntax = "proto3";
+
+package grpc;
+
+service grpc {
+    //index-client
+    rpc BatchWrite(WriteBatch) returns (Nothing);
+    rpc QueryPages(IndexQuery) returns (ReadBatch);
+
+    //storage-client
+    rpc PutChunks(Chunk) returns (Nothing);
```

Comment:
PutChunks() should accept multiple chunks not a single chunk.

Reply:
Made the necessary changes to do PutChunks multiple chunks in a single RPC call.